python/lib/client/dmod/client/request_clients.py

```diff
@@ -559,10 +614,18 @@ def extract_dataset_names(cls, response: DatasetManagementResponse) -> List[str]
         else:
             return response.data.datasets
 
-    def __init__(self, transport_client: TransportLayerClient, auth_client: Optional[AuthClient] = None, *args, **kwargs):
+    def __init__(
+        self,
+        transport_client: TransportLayerClient,
+        auth_client: Optional[AuthClient] = None,
+        *args,
+        http_client: Optional[aiohttp.ClientSession] = None,
```

Comment:
Any particular reason this was ordered this way with respect to `*args` and `**kwargs`?